src/Transposit.ts

```diff
@@ -70,6 +70,12 @@ export class Transposit {
     return JSON.parse(clientClaimJSON);
   }
 
+  private areClientClaimsValid(clientClaims: ClientClaims): boolean {
+    const expiration = new Date(clientClaims.exp * 1000);
+    const now = new Date();
```

Comment:
nit: I recently learned about `Date.now()` to get epoch millis if you wanted to avoid creating Date objects.

Reply:
Hot tip 🚒